### ISSUE TITLE
Actually implement chess patches

### ISSUE THREAD

**Issue by jackfirth** [jackfirth/chess#2]
The `chess-patch-apply` function isn't implemented yet, even though the docs claim it is. Either it should be implemented or, if the implementation won't happen for a long time, the docs should be updated to mention that it's not yet implemented.

**Comment by srfoster** [comment]
Ah.  I just found this out the hard way. :)  I came here to open an issue and found this one.

**Comment by jackfirth** [comment]
Yeah, my vacation ended before I got around to finishing this. I solemnly swear that chess patches are the only thing that looks like it's implemented but actually isn't.

**Comment by srfoster** [comment]
Lol.  I believe you.  I've been having fun with the rest.  I almost have a PGN parser implemented on top of all the non-patch stuff.

**Comment by trhopkins** [comment]
Has `chess-patch-apply` been implemented? I came here from [the Racket docs](https://docs.racket-lang.org/chess/Chess_Patches.html) to mention that the arguments `patch` and `board` out of order, and should say `board` `patch`. I apologize for improper usage, I am new to the GitHub ecosystem.

**Comment by jackfirth** [comment]
I haven't touched this at all. I likely won't, it was just a fun side project.
